src/screens/settings/blockchain-portfolio/blockchain-portfolio.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import { INavigationProps } from '../../../navigation/with-navigation-params';
+import { Text } from '../../../library';
+import { IReduxState } from '../../../redux/state';
+import stylesProvider from './styles';
+import { withTheme, IThemeProps } from '../../../core/theme/with-theme';
+import { Icon } from '../../../components/icon';
+import { smartConnect } from '../../../core/utils/smart-connect';
+import { connect } from 'react-redux';
+import { translate } from '../../../core/i18n';
+import { IBlockchainsOptions } from '../../../redux/app/state';
+import { themes } from '../../../navigation/navigation';
+import DraggableFlatList from 'react-native-draggable-flatlist';
+import { toogleBlockchainActive, updateBlockchainOrder } from '../../../redux/app/actions';
+import { Blockchain } from '../../../core/blockchain/types';
+import { ICON_SIZE } from '../../../styles/dimensions';
+import { capitalizeFirstLetter } from '../../../core/utils/format-string';
+
+export interface IReduxProps {
+    blockchains: IBlockchainsOptions;
+    toogleBlockchainActive: typeof toogleBlockchainActive;
+    updateBlockchainOrder: typeof updateBlockchainOrder;
+}
+
+const mapDispatchToProps = {
+    toogleBlockchainActive,
+    updateBlockchainOrder
+};
+
+const mapStateToProps = (state: IReduxState) => {
+    return {
+        blockchains: Object.keys(state.app.blockchains)
+            .map(key => ({ key, value: state.app.blockchains[key] }))
+            .sort((a, b) => a.value.order - b.value.order)
+    };
+};
+
+const navigationOptions = ({ theme }: any) => ({
+    title: translate('Settings.blockchainPortfolio'),
+    headerStyle: {
+        backgroundColor: themes[theme].colors.header
+    }
+});
+
+export class BlockchainPortfolioComponent extends React.Component<
+    INavigationProps & IReduxProps & IThemeProps<ReturnType<typeof stylesProvider>>
+> {
+    public static navigationOptions = navigationOptions;
+
+    public renderBlockchain(
+        item: { key: Blockchain; value: IBlockchainsOptions },
+        move: any,
+        moveEnd: any
+    ) {
+        const { styles, theme } = this.props;
+
+        return (
+            <View style={styles.rowContainer}>
+                <View style={styles.infoContainer}>
+                    <Icon name="money-wallet-1" size={ICON_SIZE} style={styles.blockchainIcon} />
+                    <Text style={styles.blockchainName}>{capitalizeFirstLetter(item.key)}</Text>
```

Comment:
you can use `textTransform: 'capitalize'` in style instead of using `capitalizeFirstLetter` function